package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "raha-blockchain",
+  "version": "0.0.1",
+  "description": "Blockchain scripts for Raha.",
+  "main": "built/index.js",
+  "author": "Raha Foundation",
+  "license": "Apache-2.0",
+  "private": false,
+  "dependencies": {
+    "bs58": "^4.0.1",
+    "buffer": "^5.1.0",
+    "node-fetch": "^2.1.2",
```

Comment:
Down the line for things like `buffer` and `node-fetch` that we do not want in a bundle sent over the wire to a browser, will these automatically be taken out?

Reply:
If it's going to the browser, we can use [`isomorphic-fetch`](https://github.com/matthew-andrews/isomorphic-fetch) instead.

Reply:
And as for [`buffer`](https://github.com/feross/buffer), it looks like Feross made it for browsers? Should work, are you worried about the filesize or something?

Reply:
I'll switch to `isomorphic-fetch`. Is buffer made by Feross? That's awesome lol. Long term I'd hope to use this module wherever we interact with the blockchain, including in the Laboratory project, so it should work in browsers too.